packages/stream-model-handler/src/model-handler.ts

```diff
@@ -90,13 +90,13 @@ export class ModelHandler implements StreamHandler<Model> {
     }
 
     const streamId = await StreamID.fromGenesis('model', commitData.commit)
-    // TODO(NET-1437): replace family with model
-    const { controllers, family } = payload.header
+    const { controllers, model } = payload.header
```

Comment:
don't you have to do `StreamID.fromBytes` like you do in model-instance-handler?

We should probably have some tests of capabilities with `Model` Streams

Reply:
correct types here, but was wondering why we are using bytes in one place (use string or streamid in tile handler before), any reason? 

I can look closer again to maybe see why

Reply:
`model` is definitely a byte array in the header: https://github.com/ceramicnetwork/js-ceramic/blob/7fa990c1a7185381743ab6f038e510ce2b28b3c9/packages/stream-model/src/model.ts#L310.

@oed  had the idea to use a byte array for model to speed up serialization/deserialization.  TileDocument doesn't use model so isn't directly relevant, and for other places that serialized StreamIDs as strings we mostly left alone to avoid breaking anything

Reply:
yup! could have sworn i saw it differ before, but must have confused it with the strings in tile stream, updated 

pr should be all set to merge now

Reply:
last week I believe they did, now they do throw, just checked (maybe model validation be added)